upcrater/src/main/java/io/crate/upcrater/Upcrater.java

```diff
@@ -257,13 +253,25 @@ private static SummaryStats execute(UpcraterConfiguration configuration, Environ
                                 e);
                         }
                     }
+
                     summaryStats.addStatusForTable(table, possibleLockId, statuses);
                 }
             }
         }
         return summaryStats;
     }
 
+    private static void markAsUpgraded(UpcraterMetaStateService metaStateService,
+                                       IndexMetaData indexMetaData,
+                                       Path indexPath) throws Exception {
+        Settings settings = Settings.builder().put(indexMetaData.getSettings())
+            .put(IndexMetaData.SETTING_VERSION_CREATED, Version.CURRENT)
+            .put(IndexMetaData.SETTING_VERSION_UPGRADED, Version.CURRENT).build();
+        IndexMetaData newIndexMetaData = IndexMetaData.builder(indexMetaData).settings(settings).build();
+        LOGGER.debug("upgrading index {}", indexMetaData.getIndex());
```

Comment:
Please start with capital and mention the **node** as we do in other log messages